### DISCUSSION TITLE
SOLVED: Line 4 of I2C 20x4 display not working

### DISCUSSION THREAD

**Discussion by gck303** [arendst/Tasmota#11043] [Q&A] [answered]
I am using a I2C 20x4 display. It is connected and will display the text, but line 4 (the bottom) is not allowing any text to display. Incidentally, a 16x2 display works perfectly, with no issue. 

The **displayrows** and **displaycols** have been manually set. 

The command **display** returns the following. Not the width and height are not correct. 
10:11:53.303 MQT: stat/esp8266/RESULT = {"Display":{"Model":1,"Width":16,"Height":2,"Mode":0,"Dimmer":2,"Size":1,"Font":1,"Rotate":0,"Refresh":2,"Cols":[20,8],"Rows":4}}

I issue the following commands and do the following...

Appears on row 1:
DisplayText[zy0]12345678901234567890

Appears on row 2:
DisplayText[zy1]12345678901234567890
DisplayText[zy3]12345678901234567890
DisplayText[zy4]12345678901234567890

Appears on row 3:
DisplayText[zy2]12345678901234567890

Is this a bug? Any suggestions on how it can be made to work? Thanks.

**Comment by gemu2015** [comment, marked as answer]
you must use l (line) and c (column) for character oriented displays, x and y is for pixel oriented displays

DisplayText[zl1c1]12345678901234567890

**Comment by gck303** [comment]
Thank you. This issue can be closed. (If I can figure out how to do it! )

I got there in the end. You just need to be sure to understand all the commands.
